multi-node/aws/.gitignore

```diff
@@ -1,4 +1,5 @@
 /artifacts/template.json
 /bin
-/clusters
-/cluster.yaml
+/assets
+/*.yaml
+*~
```

Comment:
I understand the regex & I mean for both `/*.yaml` and `*~`.

As far as *~ what files are being generated? This is not something I run into needing to be ignored.
As far as /*.yaml, what yaml files are being generated in the top level dir that you want ignored?